Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+job('e2e-tests') {
+    scm {
+        git {
+            remote {
+                github('coreos/prometheus-operator')
+                refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+            }
+            branch('${sha1}')
+        }
+    }
+
+    wrappers {
+        credentialsBinding {
+            amazonWebServicesCredentialsBinding{
+                accessKeyVariable('AWS_ACCESS_KEY_ID')
+                secretKeyVariable('AWS_SECRET_ACCESS_KEY')
+                credentialsId('Jenkins-Monitoring-AWS-User')
+            }
+            usernamePassword('QUAY_ROBOT_USERNAME', 'QUAY_ROBOT_SECRET', 'quay_robot')
+        }
+    }
+
+    triggers {
+        githubPullRequest {
+           useGitHubHooks()
+            admins(['mxinden'])
+            orgWhitelist(['coreos-inc'])
+        }
+    }
+
+    steps {
+        shell('docker build -t cluster-setup-env scripts/jenkins/.')
+    }
+
+    steps {
+        shell('docker run --privileged --rm -v $PWD:/go/src/github.com/coreos/prometheus-operator/ cluster-setup-env /bin/bash -c "cd /go/src/github.com/coreos/prometheus-operator && CGO_ENABLED=0 GOOS=linux go build --ldflags \'-extldflags \\"-static\\"\' github.com/coreos/prometheus-operator/cmd/operator"')
```

Comment:
we already discussed this offline and makes sense to only address this in a follow up PR, but we should work towards replacing this with `promu`